gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -316,26 +316,40 @@ nnsconf_loadconf (gboolean force_reload)
     /* init with 0 */
     memset (&conf, 0, sizeof (confdata));
   }
-
-  /* Read from the conf file first */
-  if (g_path_is_absolute (NNSTREAMER_CONF_FILE)) {
-    conf.conffile = g_strdup (NNSTREAMER_CONF_FILE);
-  } else {
-    /** default value of 'sysconfdir' in meson is 'etc' */
-    conf.conffile = g_build_path (G_DIR_SEPARATOR_S, root_path_prefix,
-        NNSTREAMER_CONF_FILE, NULL);
-  }
-
+#ifndef __TIZEN__
+  /** if it's not Tizen, configuration from env-var has a higher priority */
+  conf.conffile = _strdup_getenv (NNSTREAMER_ENVVAR_CONF_FILE);
   if (!g_file_test (conf.conffile, G_FILE_TEST_IS_REGULAR)) {
-    /* File not found or not configured */
     g_free (conf.conffile);
+#endif
```

Comment:
```
#ifndef __TIZEN__
  ...
  ...
    g_free (conf.conffile);
    conf.conffile = NULL;
  }
#endif
  if (conf.conffile == NULL) {
  ...
  }
```
Then, you may omit the later #ifdef/#endif, which in turn, improves the readability.

Reply:
Applied. thanks